build.nix

```diff
@@ -176,6 +194,12 @@ with rec
           echo "  ${cargoDoc}"
           ${cargoDoc}
 
+          ${lib.optionalString removeReferencesToSrcFromDocs ''
+          # Remove references to the source derivation to reduce closure size
+          find target/doc -name "*\.rs\.html" -exec \
+            perl -pi -e 's/<meta name="description" content="Source to the Rust file .*?">/<meta name="description" content="Source to the Rust file removed to reduce Nix closure size.">/' {} +
```

Comment:
I need `perl` instead of `sed` for non greedy matching. See: https://stackoverflow.com/a/1103177